types/index.d.ts

```diff
@@ -70,7 +70,7 @@ export interface RenderOptions<
    *
    *  @see https://testing-library.com/docs/react-testing-library/api/#wrapper
    */
-  wrapper?: React.ComponentType<{children: React.ReactElement}>
+  wrapper?: React.JSXElementConstructor<{children: React.ReactElement}>
```

Comment:
I'm wondering whether we should define `children` as `React.ReactNode` which seems to be the "official" type for children props. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L829

Reply:
That makes perfect sense - `ReactNode` is indeed very wide. Thanks for the explanation.